src/main/java/com/minecolonies/coremod/event/EventHandler.java

```diff
@@ -384,7 +385,7 @@ public void onBlockPlaced(@NotNull final BlockEvent.PlaceEvent event)
     {
         final EntityPlayer player = event.getPlayer();
         final World world = event.getWorld();
-        if (event.getPlacedBlock().getBlock() instanceof AbstractBlockHut)
+        if (event.getPlacedBlock().getBlock() instanceof AbstractBlockHut && event.getPlacedBlock().getBlock() != ModBlocks.blockPostBox)
```

Comment:
I think it would be better to check for the postbox below at line 397 instead, since right now it would skip the permission check upon placement.